session/resthandler.go

```diff
@@ -125,16 +125,6 @@ var (
 	dnsResolver = &dnscache.Resolver{}
 )
 
-func init() {
-	go func() {
-		t := time.NewTicker(10 * time.Minute)
-		defer t.Stop()
-		for range t.C {
-			dnsResolver.Refresh(true)
```

Comment:
Why was this removed?

Reply:
doesn't do much since it will be done on a cache fail anyways, and just increased systemcalls. Not directly related to this bugfix, but was removed during debugging and I saw no reason to re-add it.